commnad_line_rock_paper_scissors.py

```diff
@@ -6,6 +6,27 @@
 from rock_paper_scissors.game import Game
 
 
+def human_input_to_move(game, human):
+    choices = ', or '.join(
+        [
+            ', '.join(game.valid_choices[:-1]),
+            game.valid_choices[-1]
+        ]
+    )
+
+    try:
+        human.choice = game.convert_input_to_move(input('Choose {}: '.format(choices)))
+    except Exception:
+        print('Input is invalid')
```

Comment:
I was thinking the same thing about keeping human choice assignment out of the function, but this clears up where it comes to be of handy usage :slightly_smiling_face: